### DISCUSSION TITLE
Is there a way to clear the query box?

### DISCUSSION THREAD

**Discussion by lucharo** [tconbeer/harlequin#692] [Q&A, Troubleshooting, Support] [answered]
In a traditional editor, I'd go Cmd + A and then delete but in Harlequin that doesn't work

**Comment by tconbeer** [comment, marked as answer]
It does. Depending on your terminal and shell you may need to use `ctrl+a` to select all. You can also use `ctrl+w` to "close" the buffer -- if you only have 1 buffer open, it will clear it.
